modules/brew/README.md

```diff
@@ -0,0 +1,136 @@
+- Brew in `/home/linuxbrew/` is compressed in tar, copied to `/usr/share/homebrew/` & permissions to it are set to default user (UID 1000)
+- `brew-update` & `brew-upgrade` SystemD service timers are enabled (by default)
+- Brew bash & fish shell completions are copied to `/etc/profile.d/brew-bash-completions.sh` & `/usr/share/fish/vendor_conf.d/brew-fish-completions.fish`
+- `tmpfiles.d` configuration `homebrew.conf` is written with these directory locations:
+  - `/var/lib/homebrew/`
+  - `/var/cache/homebrew/`
+  - `/home/linuxbrew/`
+- `brew-setup` service is enabled
+
+### Boot-time:
+
+**`tmpfiles.d homebrew.conf`:**
+- This configuration is telling SystemD to: automatically create these necessary directories on every system boot if not available & to give them permissions of the default user (UID 1000):
+  - `/var/lib/homebrew/`
+  - `/var/cache/homebrew/`
+  - `/home/linuxbrew/`
+
+**`brew-setup`:**
+- `brew-setup` SystemD service checks if main directory used by Brew exists (`/home/linuxbrew/.linuxbrew/`)  
+  & if `brew-setup` state file exists (`/etc/.linuxbrew`)
+- If one of those paths don't exist, than Homebrew tar is extracted from `/usr/share/homebrew/homebrew.tar.zst` to `/tmp/homebrew/`
+- Extracted Homebrew is than copied from `/tmp/homebrew/` to `/home/linuxbrew/` & permissions to it are set to default user (UID 1000)
+- Temporary directory `/tmp/homebrew/` is removed
+- Empty file `/etc/.linuxbrew` is created, which indicates that brew-setup (installation) is successful & which allows setup to run again on next boot when removed
+
+**Rest of the setup:**
+- `brew-update` runs at the specified time to update Brew to the latest version
+- `brew-upgrade` runs at the specified time to upgrade Brew packages
+
+## Configuration Options
+
+### `update-interval` (optional: string, default: '6h')
+Defines how often the Brew update service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `upgrade-interval` (optional: string, default: '8h')
+Defines how often the Brew upgrade service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `auto-update` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-update.timer`.
+
+### `wait-after-boot-update` (optional: string, default: '10min')
+Time delay after system boot before the first Brew update runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `auto-upgrade` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-upgrade.timer`.
+
+### `wait-after-boot-upgrade` (optional: string, default: '30min')
+Time delay after system boot before the first Brew package upgrade runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
```

Comment:
Ok, how about:
- `auto-update`
   - need to add definition of `update` to distinguish from `upgrade`
- `update-interval`
- `wait-after-boot-update`
   - feels like this could be renamed to `update-wait-after-boot`
- `auto-upgrade`
   - need to add definition of `upgrade` to distinguish from `update`
- `upgrade-interval`
- `wait-after-boot-upgrade`
   - feels like this could be renamed to `upgrade-wait-after-boot`

Reply:
I agree with this.

I will change the code to also reflect this.

Reply:
About

`need to add definition of upgrade to distinguish from update`

Idk what would be the best suggestion.